Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -2128,27 +2128,44 @@ extension SourceKitLSPServer {
     callableUsrs += index.occurrences(ofUSR: data.usr, roles: .overrideOf).flatMap { occurrence in
       occurrence.relations.filter { $0.roles.contains(.overrideOf) }.map(\.symbol.usr)
     }
+    // callOccurrences are all the places that any of the USRs in callableUsrs is called.
+    // We also load the `calledBy` roles to get the method that contains the reference to this call.
```

Comment:
Sort of unrelated to this PR, but it'd be good to add a bunch of test cases for properties:
```
struct Foo {
  var member: Int = {
    return bar()
  }();

  func bar() {}
}
```

Call hierarchy for `bar` probably should give `member` there. `member` isn't *really* the caller, but... I think it's what people would expect to see in the UI.

Also just a regular getter:
```
struct Foo {
  var member: Int {
    return bar()
  }

  func bar() {}
}
```

Plus explicit getter/setter/etc:
```
struct Foo {
  var member: Int {
    get {
      return bar()
    }
  }

  func bar() {}
}
```

Also CC @hamishknight as he's been looking at this a bit recently (both to comment on the above and also a review of this PR in general).

Reply:
We do already have a test for valuable getters FWIW: https://github.com/apple/sourcekit-lsp/blob/4ed62392e7b4f889d44c09e4d6857a0eb28975c3/Tests/SourceKitLSPTests/CallHierarchyTests.swift#L414-L452

I agree that the rest would be good to test

Reply:
Extending that with a use of `foo` seems like a good thing to do as well.